homeassistant/components/samsungtv/media_player.py

```diff
@@ -71,13 +71,27 @@
     ):
         turn_on_action = hass.data[DOMAIN][ip_address][CONF_ON_ACTION]
         on_script = Script(hass, turn_on_action)
-    async_add_entities([SamsungTVDevice(config_entry, on_script)])
+
+    data = config_entry.data.copy()
+    while True:
```

Comment:
Why do we use a loop?

Reply:
Removed.
Not sure if the `copy()` is really needed.